modules/programs/fzf.nix

```diff
@@ -12,6 +12,16 @@ in
   options.programs.fzf = {
     enable = mkEnableOption "fzf - a command-line fuzzy finder";
 
+    defaultCommand = mkOption {
+      type = types.listOf types.str;
```

Comment:
I'm thinking this should have `type = types.nullOr types.str`. In this case it seems more natural to write "fd --type f" for the command instead of the list form.